src/conf.py

```diff
@@ -16,6 +16,33 @@
 import os
 import shlex
 
+# Call doxygen in ReadtheDocs
+read_the_docs_build = os.environ.get('READTHEDOCS', None) == 'True'
+if read_the_docs_build:
+
+    # Makes sure directory exists for doxygen output
+    cwd=os.getcwd()
+    buildpath=os.path.join(cwd,"_build")
+    if (os.path.isdir(buildpath) == 0):
+        os.mkdir(buildpath)
+    htmlpath=os.path.join(buildpath,"html")
+    if (os.path.isdir(htmlpath) == 0):
+        os.mkdir(htmlpath)
+
+    # Modify Doxyfile for ReadTheDocs compatibility
+    with open('./docs/doxygen/Doxyfile.in', 'r') as f:
```

Comment:
Could we enable the `ALPHABETICAL_INDEX` setting in our Doxyfile? I tend to use that the most to navigate the documentation.

Reply:
That is really convenient! Added into Doxyfile.in